app/boot.js

```diff
@@ -5,6 +5,7 @@ var config = require("../config/config"),
 
 module.exports = {
   es: new elasticsearch.Client({
+    apiVersion: "1.7",
```

Comment:
Hmm, from the [Amazon ES FAQ](https://aws.amazon.com/elasticsearch-service/faqs/):

> Q: Which Elasticsearch version does Amazon Elasticsearch Service support?
> 
> Amazon Elasticsearch Service currently supports Elasticsearch 1.5.2.

:( I was hoping they'd be a little more current than that...